### PR TITLE
 add offroad alerts for car unrecognized

### LINKED ISSUE #22898: Offroad alert for car unrecognized

### PR DESCRIPTION
**Pull request by adeebshihadeh**
fixes #22898

### CODE REVIEW

File: selfdrive/controls/lib/alerts_offroad.json [hunk 1]
```diff
@@ -40,5 +40,13 @@
   "Offroad_StorageMissing": {
     "text": "Storage drive not mounted.",
     "severity": 1
+  },
+  "Offroad_CarUnrecognized": {
+    "text": "openpilot was unable to identify your car. Your vehicle has new firmware versions that aren't recognized. Add new firmware to the proper vehicle. Need help? Join discord.comma.ai.",
```
Comment (adeebshihadeh) [inline selfdrive/controls/lib/alerts_offroad.json:45]: we should expand on these. perhaps we link to a wiki page or README?

Comment (pd0wm) [reply selfdrive/controls/lib/alerts_offroad.json:45]:
```suggestion
    "text": "openpilot was unable to identify your car. Your vehicle's ECUs are not recognized. Please submit a pull-request to add the firmware versions to the proper vehicle. Need help? Join discord.comma.ai.",
```

**Comment by VirtuallyChris** [comment]
I'll work on writeups for both of these situations in a Zendesk help article.

File: selfdrive/controls/lib/alerts_offroad.json [hunk 2]
```diff
@@ -40,5 +40,13 @@
   "Offroad_StorageMissing": {
     "text": "Storage drive not mounted.",
     "severity": 1
+  },
+  "Offroad_CarUnrecognized": {
+    "text": "openpilot was unable to identify your car. Your vehicle's ECUs are not recognized. Please submit a pull-request to add the firmware versions to the proper vehicle. Need help? Join discord.comma.ai.",
```
Comment (adeebshihadeh) [inline selfdrive/controls/lib/alerts_offroad.json:45]:
```suggestion
    "text": "openpilot was unable to identify your car. Your car is either unsupported or its ECUs are not recognized. Please submit a pull request to add the firmware versions to the proper vehicle. Need help? Join discord.comma.ai.",
```

File: selfdrive/controls/lib/alerts_offroad.json [hunk 3]
```diff
@@ -40,5 +40,13 @@
   "Offroad_StorageMissing": {
     "text": "Storage drive not mounted.",
     "severity": 1
+  },
+  "Offroad_CarUnrecognized": {
+    "text": "openpilot was unable to identify your car. Your car is either unsupported or its ECUs are not recognized. Please submit a pull request to add the firmware versions to the proper vehicle. Need help? Join discord.comma.ai.",
+    "severity": 0
+  },
+  "Offroad_NoFirmware": {
+    "text": "openpilot was unable to identify your car. Ensure the comma power is fully inserted in the OBD-II port of the vehicle. Need help? Join discord.comma.ai.",
```
Comment (adeebshihadeh) [inline selfdrive/controls/lib/alerts_offroad.json:49]:
```suggestion
    "text": "openpilot was unable to identify your car. Check integrity of cables and ensure all connections are secure, particularly that the comma power is fully inserted in the OBD-II port of the vehicle. Need help? Join discord.comma.ai.",
```

**Pull request merged**